pkg/api/pullrequestpayloadtest/v1/types.go

```diff
@@ -0,0 +1,48 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PullRequestPayloadTest represents the intent to run a battery of OCP release
+// payload validating jobs
+type PullRequestPayloadTest struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+
+	Spec   PullRequestPayloadTestSpec   `json:"spec"`
+	Status PullRequestPayloadTestStatus `json:"status"`
+}
+
+type ReleaseJobSpec struct {
+	Org     string `json:"org"`
+	Repo    string `json:"repo"`
+	Branch  string `json:"branch"`
+	Variant string `json:"variant,omitempty"`
+	Test    string `json:"test"`
+}
+
+type PullRequestPayloadJobSpec struct {
+	ReleaseJobSpec `json:",inline"`
+	ReleaseJobName string `json:"jobName,omitempty"`
```

Comment:
Does the name have to follow an particular format?  Does it match another system?

Reply:
It's the same name like r-c config references now, and if set, I expect it to match the `ReleaseJobSpec`. I'm not yet sure if I'll even use/need this name.